### DISCUSSION TITLE
Render images and enable sending images in the clipboard

### DISCUSSION THREAD

**Discussion by tucama** [d99kris/nchat#302] [Ideas]
this is the only functionality i miss. If this was implemented i'd ditch completely the browser app fffor nchat. Some terminals can render images like kitty and wezterm, if u could use this power in this app, it would be great

**Comment by d99kris** [comment]
Hi @tucama - I believe this request is already covered by #99 and #185 (correct me if I'm wrong) - so I will proceed to close this one, but please do upvote those other two requests. Thanks!
